core/trino-spi/src/main/java/io/trino/spi/connector/ColumnMetadata.java

```diff
@@ -190,6 +190,7 @@ private Builder(ColumnMetadata columnMetadata)
         {
             this.name = columnMetadata.getName();
             this.type = columnMetadata.getType();
+            this.defaultValue = columnMetadata.getDefaultValue();
```

Comment:
just `this.defaultValue = columnMetadata.getDefaultValue();` is enough (pushed a fix to your branch)

can you please add a test that `Builder(ColumnMetada).build()` roundtrips?